content/rancher-prime/aws/contents.lr

```diff
@@ -0,0 +1,323 @@
+title: Rancher Prime in AWS: Usage Instructions
+---
+theme: suse
+---
+body:
+
+## Prerequisites
+
+1. A Rancher-compatible EKS cluster. Please see
+   [Rancher support matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-7-6/)
```

Comment:
is there an unversioned version of this url? or do we have to update the docs for each release?